### DISCUSSION TITLE
[JIT] Classes not being updated while using vue-cli and npm run serve

### DISCUSSION THREAD

**Discussion by aymeric35** [tailwindlabs/tailwindcss#5165] [Help]
### What version of Tailwind CSS are you using?

- v2.2.7

### What build tool (or framework if it abstracts the build tool) are you using?

- postcss v7.0.36
- autoprefixer 9.8.6
- vue 2.6.11
- vue-router 3.2.0

### Reproduction repository

- https://github.com/aymeric35/vue-cli-tailwind-test

### How to reproduce the issue
Update `h1` class here: https://github.com/aymeric35/vue-cli-tailwind-test/blob/master/src/components/HelloWorld.vue#L3
Changing the color or size of the text by another value will reset the default value (like if no value was applied).

### Decription of the issue

Hello,

I wasn't sure if the Issues tab was more appropriate or not, this is the first time I'm posting on github and so I preferred to choose the discussion option rather than issue to avoid making a mistake.

My issue is that classes are not being updated while using `npm run serve` with vue-cli. Commenting and uncommenting `jit` in tailwind config file is solving the issue but doing it everytime is, as you can imagine, quite annoying.

I assume that's because vue is using postcss7, I found many threads about it on github but I couldn't solve the issue by myself because I'm using vue-cli and none of them gave a solution for vue-cli.

Also, I found this: https://tailwindcss.com/docs/just-in-time-mode#styles-don-t-update-when-saving-content-files
But I have no clue how to properly set up `TAILWIND_MODE=watch` with my current configuration in `package.json` : `"serve": "vue-cli-service serve"`
I tried adding it at the beginning or at the end but it didn't work (dependency not found if added at the end, and `'TAILWIND_MODE' is not recognized as an internal or external command operable program or batch file` if added at the beginning).

I wish all of you a good day.

**Comment by huibizhang** [comment]
Using `cross-env` package will fine.
and `TAILWIND_MODE should be add at beginning, like:

```json
"serve": "cross-env TAILWIND_MODE=watch …"
```

I also have this problem and this is my solution

https://github.com/tailwindlabs/tailwindcss/issues/5132#issuecomment-894549642

**Reply by aymeric35** [reply]
Thank you for your answer, I'll try that!
